### PR TITLE
Make map.control test pass

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: cypress/integration/map.controls.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ describe('Map controls', () => {
       expect(
         mapElt
           .getStyle()
-          .layers.find(({ id }) => id === '467f57e3ed81809ad58f090426a3893a'),
+          .layers.find(({ id }) => id === '5b0371ebc5d25ba6f7ce7068a8a959a7'),
```
Comment (mabhub) [inline cypress/integration/map.controls.js:21]: Shouldn't this string be in a variable (with a proper comment) so we know where it comes from? (and how to update it)

Comment (mabhub) [reply cypress/integration/map.controls.js:21]: cc @jrmi

Comment (CFarcy) [reply cypress/integration/map.controls.js]: Done.

Comment (mabhub) [reply cypress/integration/map.controls.js]: Perfect! 👍

**Pull request closed without merging**